spec/components/product_component_spec.rb

```diff
@@ -35,12 +35,6 @@
         )
       )
     end
-
-    it "joins the categories' name" do
-      expect(page.find('.category')).to have_content(
-        /random 1, random 2/, exact: true, normalize_ws: true
-      )
-    end
```

Comment:
Actually this is the only example in the block, so the whole block can be deleted 🧹